code/modules/mining/lavaland/weather.dm

```diff
@@ -46,7 +50,9 @@
 	update_areas()
 	for(var/mob/M in player_list)
 		if(M.z == target_z)
-			M << "<span class='danger'><B>[duration_message]</B></span>"
+			M << "<span class='userdanger'><i>[duration_message]</i></span>"
+			if(duration_sound)
+				M << duration_sound
```

Comment:
Have you tested how this sounds in game?

What happens if the sound plays and they leave the area?